src/accessories/CleanUpHelper.ttslua

```diff
@@ -185,6 +186,13 @@ end
 -- modular functions, called by other functions
 ---------------------------------------------------------
 
+function triggerSoundCube()
+  local soundCube = getObjectsWithTag("SoundCube")[1]
+  if soundCube then
+    soundCube.AssetBundle.playTriggerEffect(0)
```

Comment:
Trying to remember (and avoid changes to) these indexes is going to be fragile.  Suggest:
- Add code and an Api to the SoundCube with "playVacuum", "tollTheBell", etc
- Reference the Api and use those methods

Reply:
Yeah, that is a good idea (and I thought of it too after I created the PR 😅)